api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -708,24 +712,114 @@ default void openBook(final @NotNull Book book) {
   // ------------------------
 
   /**
-   * Sends a resource pack request to this audience.
+   * Sends a request to apply resource packs to this audience.
+   *
+   * <p>Multiple resource packs are only supported since 1.20.3. On older versions, all requests behave as if {@link ResourcePackRequest#replace()} is set to {@code true}.</p>
```

Comment:
I wonder if we want to do something different here. Nobody is realistically going to want to set a bunch of packs with replace to true. I think we should just emulate this behaviour by only applying the final/first pack instead (effectively the same, but without the pointless network load/etc.

Reply:
I'd imagine that the only place in which replace would be true is for the first pack in the stack, "reset and apply these", assuming that the client supports that behavior (I would hope so)
as for legacy versions, I think that adventure should just recommend for a behavior here, i.e. impls should only care about grabbing and sending the last pack in the request?

assuming I'm not barking up the wrong tree for the discussion here

Reply:
`replace` applies to the overall request, not to individual packs within the request here -- that's the distinction between `ResourcePackInfo` and `ResourcePackRequest` really.

But yeah, the comments about legacy versions really only apply to whoever is masochistic enough to implement the resource pack API in -platform, modern impls can just ignore that bit.